### DISCUSSION TITLE
I don't know how to put foreignTable in Flutter

### DISCUSSION THREAD

**Discussion by iCodePoet** [supabase/supabase#6312] [Questions]
Hello, I don't know how to put foreignTable in Flutter. Does anyone know?

![image](https://user-images.githubusercontent.com/29420674/161901799-a5f846e5-5010-4dad-baeb-de0e8d5c827c.png)

**Comment by dshukertjr** [comment]
Hi @Jay-flow 

So sorry for the late response. Filtering the foreign table using `or` is currently not yet supported on Flutter SDK. 

The good news is you can still use the `filter()` method to achieve this. Here is an example:

```dart
final res = await supabase
    .from('countries')
    .select('id, cities(*)')
    .filter('cities.name', 'or', '("Paris","Tokyo")')
    .execute();

```

**Reply by iCodePoet** [reply]
Thx 👍🏼 @dshukertjr 
Is there any plan to add this function to Flutter?

**Reply by dshukertjr** [reply]
@Jay-flow Yes, we would love to cover every feature that the javascript SDK is covering. Would you mind opening an issue [here](https://github.com/supabase-community/supabase-flutter)?

**Reply by iCodePoet** [reply]
@dshukertjr Sure! I opened an issue in the repo
https://github.com/supabase-community/supabase-flutter/issues/121
